cowprotocol/cowswap
# feat: create library for widget-react

# Summary

Creates a new React component library, publishable in NPM, that makes it easier to create React components.

<img width="2047" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/dbf4e523-050d-487d-a26a-0667f62f8579">

NPM: https://www.npmjs.com/package/@cowprotocol/widget-react 


# To Test
* Test is this PR https://github.com/cowprotocol/widget-examples/pull/1
* Also check out: https://swap-dev-git-npm-react-widget-cowswap.vercel.app/#/kitchen-sink?chain=mainnet
